indexer/osm_editor.cpp

```diff
@@ -0,0 +1,109 @@
+#include "osm_editor.hpp"
+
+#include "indexer/feature_decl.hpp"
+#include "indexer/index.hpp"
+
+#include "std/map.hpp"
+#include "std/set.hpp"
+
+namespace osm
```

Comment:
We don’t have such a rule in our repository, and never had it. And I don’t see any reason to have it.
For example, osm namespace now is used in opening_hours and generator.
Namespaces covering, m2, math etc are used in geometry library.

Any code located in the module can use any namespace which helps to understand this code purpose. Now there is no many osm-only related lines of code in these earliest commits, but it is definitely coming in next commits to the osm::Editor class in about a week.